diff --git a/node_modules/super-three/build/three.module.js b/node_modules/super-three/build/three.module.js
index 3aea48d..c73c860 100644
--- a/node_modules/super-three/build/three.module.js
+++ b/node_modules/super-three/build/three.module.js
@@ -26167,7 +26167,16 @@ class WebXRManager extends EventDispatcher {
 
 			const cameras = cameraXR.cameras;
 			var object = poseTarget || camera;
-			const parent = object.parent;
+			// const parent = object.parent;
+			// Object is the Group Three.js node that contains the virtual camera.  In 1.3.0, the
+			// original code for getting the parent is:
+			//   const parent = object.parent;
+			// In 1.2.0, the code for setting the parent local variable is:
+			//   const parent = camera.parent
+			// Therefore, in 1.3.0 the parent was the scene, where as in 1.2.0 the parent is the Group
+			// Three.js node.  We want to continue the 1.2.0 behavior in order for xr-aframe to work
+			// with both 1.2.0 and 1.3.0.
+			const parent = camera.parent;
 
 			updateCamera( cameraXR, parent );
 
@@ -27766,6 +27780,8 @@ function WebGLRenderer( parameters = {} ) {
 		info.programs = programCache.programs;
 
 		_this.capabilities = capabilities;
+		_this.attributes = attributes;
+		_this.geometries = geometries;
 		_this.extensions = extensions;
 		_this.properties = properties;
 		_this.renderLists = renderLists;
